scripts/release/mule/test/test.sh

```diff
@@ -30,8 +30,8 @@ export OS_TYPE
 
 export SHA
 
-ALGORAND_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-beta || echo algorand)
-DEVTOOLS_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-devtools-beta || echo algorand-devtools)
+ALGORAND_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-beta || [ "$CHANNEL" = alpha ] && echo algorand-alpha || echo algorand)
+DEVTOOLS_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-devtools-beta || [ "$CHANNEL" = alpha ] && echo algorand-devtools-alpha || echo algorand-devtools)
```

Comment:
can we use parenthesis to explicitly show groups instead of implicitly relying on priority order of operands please?